public/index.html

```diff
@@ -50,12 +64,126 @@
             {disabled: false, country: undefined},
             {disabled: false, country: undefined},
             {disabled: false, country: undefined}
+          ],
+          AS: {
+            id: [214, 254, 192],
+            isFetched: false,
+            isFetching: false,
+            didFailed: false
+          }
```

Comment:
We will not need this any more as we don't know what ASNs we should render and are not able to pass it in the views. We can easily delete